hack/lib/utilities.sh

```diff
@@ -218,6 +218,10 @@ capact::install() {
       CAPACT_OVERRIDES+=",hub-public.populator.manifestsLocation.branch=${HUB_MANIFESTS_SOURCE_REPO_REF}"
     fi
 
+    if [ -n "${HUB_MANIFESTS_SOURCE_REPO_URL:-}" ]; then
```

Comment:
Please add `HUB_MANIFESTS_SOURCE_REPO_URL`  to the list of accepted variables at the top of the function.

With current approach we can not override is it always use the var from consts.sh is this expected?

Reply:
that's a good point, probably we should allow to override it. Changes introduced to `workflow` are not used so the `HUB_MANIFESTS_SOURCE_REPO_URL` will be still set for the old commit though.